### DISCUSSION TITLE
What do "extra_cost" and "LHS" do?

### DISCUSSION THREAD

**Discussion by ghramsey** [MattFiler/LegendPlugin#131] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/51859157/211216167-57778b04-075d-413c-b283-782bc6766bf7.png)

What does this parameter extra_cost do?
It's a numerical entry and seems to be float since a decimal point can be added.
I don't see where it's referenced to in the door package archetype.
And what is "LHS" under several of the door references? 
It's an abreviation I cannot find.

**Comment by MattFiler** [comment, marked as answer]
LHS is usually paired with RHS, which will evaluate to the left and right hand side of a numerical calculation. For example, FloatGreaterThan will evaluate if the value on the left is greater than the one on the right or vice versa.

The cost value in regards to pathfinding will be to do with how the AI picks its pathing through an area. The wait node I assume is adding extra value to that section of the navmesh, which would weight it lower in the AI's choice of where to route.

**Reply by ghramsey** [reply]
Interesting.  I'm still trying to find how to make the aliens know they cannot path thru the closed transit office door so they upvent.
I'll play w/ the values some more.
